### PR TITLE
Adds sponsors API to events/view route

### LINKED ISSUE #550: Implement sponsors API on events/<event-id> route
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

image

### PR DESCRIPTION
**Pull request by geekyd**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Proposed changes in this PR
- Integrates sponsors API to events/view route

Fixes #550

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-649-open-event-frontend-fossasia.surge.sh

File: app/controllers/events/view/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    {
+      propertyName   : 'type',
+      title          : 'Type',
+      disableSorting : true
+    },
+    {
+      propertyName   : 'level',
+      title          : 'Level',
+      disableSorting : true
```
Comment (CosmicCoder96) [inline app/controllers/events/view/index.js]: @geekyd They should be sortable by level right ?

**Comment by niranjan94** [comment]
Deployment link: https://pr-649-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-649-open-event-frontend-fossasia.surge.sh

**Pull request merged**